superdesk/notification.py

```diff
@@ -15,9 +15,9 @@
 import asyncio
 import websockets
 
-from flask import json
 from flask import current_app as app
 from datetime import datetime
+from bson.json_util import dumps
```

Comment:
I used this to serialize the ObjectId for the user and the desk. For flask.json we'd need an encoder to handle the ObjectId but with this one it is built in.